trust-store-trust-policy-specification.md

```diff
@@ -135,6 +154,13 @@ Value descriptions
 - **`warn`**: This means implementation MUST perform the validation and if validation fails(because of any reason) the implementation MUST log an error and MUST NOT fail validation.
 - **`skip`**: This means implementation MUST NOT perform the validation.
 
+#### Scope Constraints
+
+- There MUST only be one trust policy applicable to an artifact.
+- There MUST not be two trust policies with the same scope.
```

Comment:
Does this mean I need to include all keys trusted for a scope/namespace in the same policy? I'm imagining people would want an easy way to say "update the key for the security department" separate from the "update the key for the CI server" since they will probably be updated by separate teams on their own schedule.

Reply:
Yes and no. Yes user will need to reference all trust-store in one trust-policy that's applicable to that scope
and as its a reference to trust store, different team can update there trust-store independently.